src/main/java/org/springframework/samples/petclinic/rest/importcsv/ImportCSV.java

```diff
@@ -26,114 +28,239 @@
     @Autowired
     private ClinicService clinicService;
 
+    private static final int COMMAND_FIELD_NUMBER = 5;
```

Comment:
I like that you created variables to ease configuration - since the parameters are private, javadoc is not mandatory in my opinion